packages/flutter/lib/src/material/chip.dart

```diff
@@ -7,6 +7,8 @@ import 'package:flutter/widgets.dart';
 import 'colors.dart';
 import 'icon.dart';
 
+const double _kSize = 32.0;
```

Comment:
Some names might be useful here:
const double _kChipHeight = 32.0;
const double _kAvatarDiamater = _kChipHeight;